setup.cfg

```diff
@@ -2,7 +2,8 @@
 exclude = .eggs,ENV,build,docs/conf.py,venv
 
 [yala]
-linters=pylint,pycodestyle,isort
```

Comment:
@rmotitsuki can you add this `linters=pylint,pycodestyle,isort` back? It turns out `yala` can fallback to existing installed linters, so having this explicit configured avoid surprises and false positives in case where it's running there are additional installed linters.